api/src/main/java/ca/bc/gov/educ/api/distribution/process/BaseProcess.java

```diff
@@ -129,10 +130,21 @@ protected void mergeDocumentsPDFs(ProcessorData processorData, String mincode, S
             //Naming the file with extension
```

Comment:
Looks like there might be a way to run in optimized resource mode (Sumathi found this). Try adding the following:
pdfMergerUtility.setDocumentMergeMode(PDFMergerUtility.DocumentMergeMode.OPTIMIZE_RESOURCES_MODE);